src/pages/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { useHistory, Switch, Route } from 'react-router-dom';
+import { DappMetaData } from 'types';
+import { importAll } from 'utils';
+import Web3 from 'web3';
+import Head from 'layout/Head';
+import Web3Wrapper from 'containers/Web3Wrapper';
+import App from './App';
+import NotFound from './NotFound';
+
+export default () => {
+  const [meta, setMeta] = React.useState<DappMetaData>({
+    tokenName: '',
+    tokenSymbol: '',
+    tokenOwnerWebsite: '',
+    siteUrl: '',
+    totalPresale: '0',
+    referralBP: '0',
+    basisPoint: '0',
+    accountCap: '0'
+  });
+
+  const [favicon, setFavicon] = React.useState<string>('');
+
+  const [showError, setShowError] = React.useState<boolean>(false);
+
+  const history = useHistory();
+
+  React.useEffect(() => {
+    const fetchProjectMeta = async () => {
+      try {
+        let config: DappMetaData = meta;
+        const project: string = history.location.pathname.split('/')[1];
+
+        let images = importAll(
```

Comment:
I think we don't need to use this pattern. I added it as testing purpose

Reply:
What pattern do we need?

Reply:
I mean we don't need the `importAll` function since we use the ipfs object url